src/Core/Hook/RenderedHook.php

```diff
@@ -66,4 +66,18 @@ public function getContent()
     {
         return $this->content;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function outputContent()
+    {
+        $output = '';
+
+        foreach ($this->content as $partialContent) {
+            $output .= array_values($partialContent)[0];
```

Comment:
Don't care about all values `array_slice($partialContent, 0, 1)`

Reply:
Ok I'll add another unit test for that

Reply:
when I did that, I've find out that the return of HookDispatcher:dispatchRendering() was not in the expected form.

I don't want to override the results that come from the Adapter as it keep the compatibility with existing behavior, so I've formatted a little bit the expected content to be compliant with RenderedHook:$content signature.